portal/app/models.py

```diff
@@ -174,3 +174,14 @@ class Meta:
     farm_device = models.ForeignKey(FarmDevice, null=True)
     device_data = JSONField(null=True, blank=True, default=None)
     timestamp = models.DateTimeField(default=timezone.now)
+
+
+class FarmMaxDemand(models.Model):
+
+    class Meta:
+        db_table = 'farm_max_power_demand'
+
+    home = models.ForeignKey(Home)
+    max_power = models.FloatField(default=0)
+    month_pst = models.IntegerField(default=0)
```

Comment:
why would month need `pst`? What does that even mean?

Reply:
because if Im in the last day of the month and the server is utc, if I make a call after 5pm PST time the server will give the next month and it will make Pmax0 = 0. But the system is running in CA thus PST and it's still not technically the next month locally

Reply:
Hummm, ok, makes sense I guess